common-lib/src/main/scala/com/gu/mediaservice/lib/argo/ArgoHelpers.scala

```diff
@@ -24,6 +24,27 @@ trait ArgoHelpers extends Results {
     serializeAndWrap(response, Ok)
   }
 
+  def respondCollections[T](
+    uri: Option[URI] = None,
+    data: Map[String, Seq[T]],
+    links: List[Link] = Nil
+  ) (implicit writes: Writes[T]): Result = {
+    val collections = data.map({ case (key, collection) => {
+      key -> CollectionReponse(
```

Comment:
Convenience for the client. They will get a `uri` if there's a reason to have separate endpoints for pending and published sets.